data/metadata.json

```diff
@@ -0,0 +1,5823 @@
+{
```

Comment:
Values for `type`, `minLength`, `pattern`, `default` and `examples` can also be read from the json schema. Should I remove this redundancy here?

Reply:
It looks like I can't find that file anymore. We should probably discuss that Wednesday :thinking:

Reply:
Seems like github isn't able to follow up on a rebase I did to keep a clean commit history. Sorry for that. In [this view](https://github.com/secvisogram/secvisogram/pull/255/files), you should be able to see the latest diffs, still.

Reply:
I removed the redundant fields in 046fd9febe420bf13e5c14a3370eabb2448b1f8e .